docs/content/docs/deployment/filesystems/azure.md

```diff
@@ -83,4 +103,19 @@ environment variable `AZURE_STORAGE_KEY` by setting the following configuration
 fs.azure.account.keyprovider.<account_name>.blob.core.windows.net: org.apache.flink.fs.azurefs.EnvironmentVariableKeyProvider
 ```
 
+### ABFS
+
+Hadoop's ABFS Azure Filesystem supports several ways of configuring authentication. Please visit the [Hadoop ABFS documentation](https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html#Authentication) documentation on how to configure.
+
+{{< hint info >}}
+Azure recommends using Azure managed identity to access the ADLS Gen2 storage accounts using abfs. Details on how to do this are beyond the scope of this documentation, please refer to the Azure documentation for more details.
+{{< /hint >}}
```

Comment:
Anything that helps the user to setup the filesystem correctly without trial and error is always good. However, I'd be completely fine in doing that in a follow-up PR; for me, it doesn't seem strongly connected to this change (but I could be wrong).

Reply:
Thanks for the links @izzyacademy Instead of adding the list of Azure services directly in the Flink documentation which may get outdated, I added links to the page. I hope this looks ok.